modules/monitoring-configuring-alertmanager-to-send-notifications-for-user-defined-alert-routing.adoc

```diff
@@ -3,10 +3,10 @@
 // * observability/monitoring/managing-alerts.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="applying-a-custom-configuration-to-alertmanager-for-user-defined-alert-routing_{context}"]
-= Applying a custom configuration to Alertmanager for user-defined alert routing
+[id="configuring-alertmanager-to-send-notifications-for-user-defined-alert-routing_{context}"]
+= Configuring the Alertmanager to send notifications for user-defined alert routing
```

Comment:
"send notifications for [...] alert routing" doesn't click for me.

```suggestion
= Configuring Alertmanager routing for user-defined alerts
```
or

```suggestion
= Configuring Alertmanager for user-defined alerts
```

or

```suggestion
= Configuring notifications for user-defined alerts
```